src/core/Board.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import { Card } from "common/components";
+import { CardFaces } from "common/components/CardFaces";
+
+import styles from "./Board.module.scss";
+
+export const Board = () => {
+  const doubleCards = arr => {
+    return arr.concat(arr);
+  };
+
+  //https://stackoverflow.com/questions/2450954/how-to-randomize-shuffle-a-javascript-array
+  const shuffle = arr => {
```

Comment:
While you may not need to reuse `shuffle` in this project, on a larger project you may want to use it elsewhere. To make this function reusable, you can add create a `shuffleArray.js` file inside of `common/utils`.

Reply:
Really happy to see you referencing where the code was borrowed from. Great practice!